ncclient/devices/sros.py

```diff
@@ -0,0 +1,54 @@
+from lxml import etree
+
+from .default import DefaultDeviceHandler
+from ncclient.operations.third_party.sros.rpc import MdCliRawCommand
+from ncclient.xml_ import BASE_NS_1_0
+
+
+def passthrough(xml):
+    return xml
+
+class SrosDeviceHandler(DefaultDeviceHandler):
+    """
+    Nokia SR OS handler for device specific information.
+    """
+
+    def __init__(self, device_params):
+        super(SrosDeviceHandler, self).__init__(device_params)
+
+    def get_capabilities(self):
+        """Set SR OS device handler client capabilities
+
+        Set additional capabilities beyond the default device handler.
+
+        Returns:
+            A list of strings representing NETCONF capabilities to be
+            sent to the server.
+        """
+        base = super(SrosDeviceHandler, self).get_capabilities()
+        additional = [
+            'urn:ietf:params:xml:ns:netconf:base:1.0',
```

Comment:
This is already in the base capabilities of the `DefaultDeviceHandler` and will cause a duplicate.

Reply:
This is actually not a duplicate - RFC4741 initially registered 2 base NETCONF namespaces of which this defined one is in the IANA IETF XML registry - https://www.iana.org/assignments/xml-registry/xml-registry.xhtml#ns

```
urn:ietf:params:netconf:base:1.0
urn:ietf:params:xml:ns:netconf:base:1.0
```

The first is rather defined in the `DefaultDeviceHandler`